src/components/SearchInput.vue

```diff
@@ -289,4 +319,12 @@ em.hilite {
   /* font-style: normal; */
   font-weight: bold;
 }
+
+.input-group-field.category-list {
+  flex-grow: 1;
```

Comment:
I've never used flexbox either, but it seems like it would make various layout tasks a lot easier. In this case, it's used by the `input-group-field` class, and I need to override it so that the fields aren't equal size.